includes/hub/admin/class-membership-plans.php

```diff
@@ -75,16 +75,15 @@ public static function render() {
 	 * @param \Newspack_Network\Node\Node $node The node.
 	 * @param string                      $collection_endpoint The collection endpoint.
 	 * @param string                      $collection_endpoint_id The collection endpoint ID.
+	 * @param array                       $query_args The query args.
 	 */
-	public static function fetch_collection_from_api( $node, $collection_endpoint, $collection_endpoint_id ) {
-		$endpoint = sprintf( '%s/wp-json/%s', $node->get_url(), $collection_endpoint );
-		if ( Network_Admin::use_experimental_auditing_features() ) {
-			$endpoint = add_query_arg( 'include_active_members_emails', 1, $endpoint );
-		}
+	public static function fetch_collection_from_api( $node, $collection_endpoint, $collection_endpoint_id, $query_args = [] ) {
+		$endpoint = add_query_arg( $query_args, sprintf( '%s/wp-json/%s', $node->get_url(), $collection_endpoint ) );
 		$response = wp_remote_get( // phpcs:ignore
 			$endpoint,
 			[
 				'headers' => $node->get_authorization_headers( 'get-woo-' . $collection_endpoint_id ),
+				'timeout' => 60, // phpcs:ignore
```

Comment:
nonblocking nit: Does the default not work here?

Reply:
What do you mean? Default timeout is `5`.

Reply:
Ah. Whoops. For some reason I thought this was `30`. Nevermind!